### PR TITLE
[skip ci] Add additional maintainers

### PR DESCRIPTION
**Pull request by alippai**
Releases are lagging behind 2+ months now. Adding extra maintainers to ensure smooth release process. 

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by alippai** [comment]
@conda-forge/core can you approve, please? I'm following [Taking over Abandoned Feedstock](https://github.com/PrernaSingh587/conda-forge.github.io/blob/patch-10/src/maintainer/updating_pkgs.rst#taking-over-abandoned-feedstock)

**Comment by alippai** [comment]
Reference: https://github.com/conda-forge/sentry-sdk-feedstock/pull/79#issuecomment-808474714

File: recipe/meta.yaml [hunk 1]
```diff
@@ -46,4 +46,6 @@ about:
 
 extra:
   recipe-maintainers:
+    - alippai
+    - dgasmith
```
[the remark this answers is not in the document]
Comment (alippai) [reply recipe/meta.yaml:50]: @beckermr Thanks! @dgasmith commented on the [previous release PR](https://github.com/conda-forge/sentry-sdk-feedstock/pull/79#issuecomment-808474714), that's why I added him. I don't know him, so we'll see whether he's still happy to take on the maintainership.

[the remark this answers is not in the document]
Comment (dgasmith) [reply recipe/meta.yaml:50]: Happy to help with maintainership of this repo- thank you for reaching out!

**Pull request merged**